### PR TITLE
Remove xunit dependancy

### PR DESCRIPTION
**Pull request by joycinho26**
Xunit dependancy has been removed

### CODE REVIEW

**Review by vliashko — CHANGES_REQUESTED** [review]
Merge from develop / resolve conflicts

File: BankingSystem.API/BankingSystem.API.sln
Comment (vliashko) [inline BankingSystem.API/BankingSystem.API.sln:1]: Why you add this file? We already have sln file

File: BankingSystem.API/BankingSystem.API.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
```
Comment (vliashko) [inline BankingSystem.API/BankingSystem.API.csproj]: ItemGroup is not needed any more

File: BankingSystem.API.Tests/BankingSystem.API.Tests.csproj [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
+
```
Comment (vliashko) [inline BankingSystem.API.Tests/BankingSystem.API.Tests.csproj]: Not needed changes

**Review by vliashko — CHANGES_REQUESTED** [review]

File: BankingSystem.API/BankingSystem.API.sln
Comment (joycinho26) [reply BankingSystem.API/BankingSystem.API.sln:1]: i deleted now ,it was a mistake

File: BankingSystem.API/BankingSystem.API.csproj (hunk 1 above)
Comment (joycinho26) [reply BankingSystem.API/BankingSystem.API.csproj]: okay ,done!

**Review by vliashko — APPROVED** [review]

**Pull request merged**